fs/ftpfs.py

```diff
@@ -96,16 +96,14 @@ def manage_ftp(ftp):
     finally:
         try:
             ftp.quit()
-        except:  # pragma: nocover
+        except:  # pragma: no cover
             pass
 
 
 def _parse_ftp_error(error):
-    # type: (ftplib.Error) -> Tuple[Union[int, Text], Text]
+    # type: (ftplib.Error) -> Tuple[Text, Text]
```

Comment:
The code doesn't actually need to be an integer. We only ever test for equality. Simplifies things a little.